packages/kbn-dom-drag-drop/src/providers/announcements.tsx

```diff
@@ -10,11 +10,7 @@ import { i18n } from '@kbn/i18n';
 import { DropType } from '../types';
 import { HumanData } from '.';
 
-type AnnouncementFunction = (
-  draggedElement: HumanData,
-  dropElement: HumanData,
-  announceModifierKeys?: boolean
```

Comment:
this param is actually not needed - it was added when the implementation was different. Basically it checks if we want to add the information to the user about the extra keys (eg 'shift to swap') and if the key is pressed (so swap is highlighted) then didn't inform the user about the keys. I think it's better to always inform the user about modifier keys so I removed it.